installer/manifests/keptn/templates/api-gateway-nginx.yaml

```diff
@@ -374,7 +374,7 @@ spec:
       terminationGracePeriodSeconds: {{ .Values.apiGatewayNginx.gracePeriod | default 60 }}
       containers:
         - name: api-gateway-nginx
-          image: {{ .Values.apiGatewayNginx.image.repository }}:{{ .Values.apiGatewayNginx.image.tag }}
+          image: {{ include "service.image" ( dict "imageRoot" .Values.apiGatewayNginx.image "global" .Values.global.apiGatewayNginx "defaultTag" .Values.global.apiGatewayNginx) }}
```

Comment:
`.Values.global.apiGatewayNginx` does not exist in the values file :)
I would also say that the `defaultTag` here could also come directly from the `apiGatewayNginx` section in the values file.

Reply:
Line 88 in the values file
https://github.com/keptn/keptn/pull/8152/files#diff-cdc92c80acd2d242526b0c626f706e98657d10caa29224b7ac05fea91dddfd4cR88